plugins/modules/aci_l3out_static_routes.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l3out_static_routes
+short_description: Manage External Subnet objects (l3extSubnet:extsubnet)
+description:
+- Manage External Subnet objects (l3extSubnet:extsubnet)
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  logical_node:
+    description:
+    - Name of an existing logical node profile.
+    type: str
+  fabric_node:
+    description:
+    - Name of an existing fabric node.
+    type: str
+  static_route:
```

Comment:
rename to "prefix" (we can add static_route and route into the aliases)

Reply:
@lhercot Should Static_route be changed to prefix or prefix_route?